saltyfacu/ape-tax
# Feature/Multicall vault

## What it does ✨
Replaced the `Promise.all` part of the code by a single multicall to perform 1 single call to init all the vault information on the vault pages.

## How to test ✅
Has been tested with all the networks.
You can just navigate and try to use the app. All the data should correctly load.

## Note 📝
The strategy part is not in a multicall situation right now. An update of ethcall should be deployed and we, then, will be able to update this part too
